pom.xml

```
@@ -4,7 +4,7 @@
 
     <groupId>com.sandflow</groupId>
     <artifactId>regxmllib</artifactId>
-    <version>1.1.4</version>
+    <version>1.1.5-rc.1</version>
```

1.1.5 was already [released](https://github.com/sandflow/regxmllib/releases/tag/1.1.5), I believe this should be 1.1.6-rc.1:

```suggestion
    <version>1.1.6-rc.1</version>
```